ADMIN.adoc

```diff
@@ -0,0 +1,35 @@
+= Repository Administrator Responsibilities
+
+== Administrator rotation schedule:
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be administrator is voluntary.
+
+== Administrator responsibilities:
+
+As the administrator, you are in charge of performing the following duties:
+
+=== Being the Point of Contact for general issues related to the Supplementary Style Guide repository
+
+* You are the point of contact in case other contributors experience issues with the repository.
+When people reach out to you about an issue, try to fix it. If you cannot fix the issue, report it to a member of the initiative who can.
+
+=== Maintaining the Style Guide site on GitHub
+
+* Ensure that the link:https://redhat-documentation.github.io/supplementary-style-guide/[style guide site] is available and displayed correctly.
+* Create pull requests to update the link:https://github.com/redhat-documentation/supplementary-style-guide/blob/master/README.md[README.md], link:https://github.com/redhat-documentation/supplementary-style-guide/blob/master/GUIDELINES.adoc[GUIDELINES.adoc] and the link:https://github.com/redhat-documentation/supplementary-style-guide/blob/master/CONTRIBUTING.md[Contribution guide], if necessary.
+
+=== Driving resolution of link:https://github.com/redhat-documentation/supplementary-style-guide/issues[open issues]
+
+* Review link:https://github.com/redhat-documentation/supplementary-style-guide/issues[issues] submitted to the Supplementary Style Guide GitHub repository.
+* Ensure that discussion on open issues is moving forward consistently. If the discussion does not progress, notify link:https://github.com/orgs/redhat-documentation/teams/ccs-style-council[CCS Style Council] members and report any issues that need your peer's attention at the next meeting.
+* Update issue priority labels according to decisions made in meetings.
```

Comment:
I would probably remove this, I think this might confuse people more than help. We don't really have priority labels (like low, medium, high). The only thing we have really is the "on hold", which I don't think we need admin rotation members to worry about adding.

Reply:
+1.

Reply:
@bergerhoffer That's reasonable. Thank you. I removed the line per your suggestion:)